web/packages/teleterm/src/ui/Search/pickers/ResultList.tsx

```diff
@@ -115,35 +111,29 @@ export function ResultList<T>(props: ResultListProps<T>) {
         )}
       </Separator>
       <Overflow role="menu">
+        {ExtraComponent}
         {items.map((r, index) => {
           const isActive = index === activeItemIndex;
           const { Component, key } = props.render(r);
 
           return (
-            <StyledItem
+            <InteractiveItem
               ref={isActive ? activeItemRef : null}
               role="menuitem"
               $active={isActive}
               key={key}
               onClick={() => props.onPick(r)}
             >
               {Component}
-            </StyledItem>
+            </InteractiveItem>
           );
         })}
-        {shouldShowNoResultsCopy && NoResultsComponent}
       </Overflow>
     </>
   );
 }
 
-const StyledItem = styled.div`
-  &:hover,
-  &:focus {
-    cursor: pointer;
-    background: ${props => props.theme.colors.levels.elevated};
-  }
-
+export const NonInteractiveItem = styled.div`
```

Comment:
Noninteractive items cannot be `active` AFAIK so we could move that `background` part to `InteractiveItem`. What's the convention behind adding a dollar sign to props?